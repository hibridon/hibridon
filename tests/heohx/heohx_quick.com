inp=heohx_quick.inp
job=heohxq
run
printc
difcrs,heohxq,1,1,2,1,0,180,0.5,1,55
exit
