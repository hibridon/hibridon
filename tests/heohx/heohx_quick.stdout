
 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V xdate

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
SYS=Debian bullseye/sid ( 5.4.0-80-generic); MACH=
CC=; FC=



 --------------------------------------------------------------------------
 Hibridon>  Hibridon>  Hibridon>  ** J =    0 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:08:35 2021
 ** J =    1 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:08:35 2021
 ** J =    2 JLPAR = 1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:08:36 2021
 ** J =    0 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:08:36 2021
 ** J =    1 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:08:36 2021
 ** J =    2 JLPAR =-1 FINISHED;  CPU: 00:00:00   WALL: 00:00:00   DATE: Aug 12 14:08:36 2021
 ===============================================================================
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:    70
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    NaN %
      CURRENT DATE:  Aug 12 14:08:36 2021
 ===============================================================================
 Hibridon> 
 INTEGRAL CROSS SECTIONS READ FROM FILE Heohxq1.ics                             
 WRITTEN:   Aug 12 14:08:36 2021
 LABEL:     R. TOBOLA AND J.KLOS He-OH(X) RCCSDT 11A        
 POT NAME:  Cybulski et al He-OH(X) RCCSDT PES              

 ** INTEGRAL CROSS SECTIONS; IEN= 1 **
    RMU=   3.2399  E=   100.00  JLPAR=-1  JTOT-1=  0.5  JTOT-2=   2.5  JTOT-D=  1
 ** CC CALCULATION, JLPAR=-1 **

 ** COLUMN HEADINGS ARE INITIAL STATES, ROW HEADINGS ARE FINAL STATES **

           J=   1.5       2.5       1.5       2.5
   J    I | I=   -1        -1         1         1
 
  1.5   -1   3.122E+00  1.136E-01  3.725E-02  1.080E+00
  1.5    1   3.727E-02  3.379E-02  3.511E+00  3.174E-01
  2.5   -1   2.731E-02  1.932E+01  8.117E-03  4.046E-02
  2.5    1   2.630E-01  4.097E-02  7.722E-02  1.836E+01
 Hibridon> 
**  DIFFERENTIAL CROSS SECTION (ANG^2/SR)
    TODAYS DATE:  Aug 12 14:08:36 2021
    S-MATRICES READ FROM FILE Heohxq1.smt                             
      WRITTEN:   Aug 12 14:08:35 2021
      LABEL:     R. TOBOLA AND J.KLOS He-OH(X) RCCSDT 11A        
      POT NAME:  Cybulski et al He-OH(X) RCCSDT PES              
    E-TOT:      100.000;  REDUCED MASS:    3.240    
    TRANSITION:  J1=   1 IN1=   1  ->  J2=   2 IN2=   1
    SUMMING PARTIAL WAVES FROM JTOT = 0 TO   2
    WAVE VECTOR IN INITIAL CHANNEL:      2.3199 Bohr^(-1);     4.3839 Angstroms^(-1)
    COLLISION ENERGY:   100.0000 cm-1;    12.3985 meV
    DIFFERENTIAL CROSS SECTIONS SAVED IN FILE: Heohxq1.dcs

 ** DIFFERENTIAL CROSS SECTION CALCULATION FINISHED:
    ELAPSED TIME: 00:00:00   CPU TIME:  00:00:00 
 Hibridon> 